packages/terra-clinical-data-grid/src/terra-dev-site/doc/clinical-data-grid/DatagridExamples.2/DatagridWithCustomContent.jsx

```diff
@@ -115,7 +115,7 @@ class DatagridWithCustomContent extends React.Component {
 
   render() {
     return (
-      <div style={{ height: '800px' }}>
+      <div className={cx('data-grid-div-style')}>
```

Comment:
we can keep the name as 'data-grid-parent-div-style'

Reply:
Classnames are renamed.